packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/CreateLoanScheme.test.ts

```diff
@@ -0,0 +1,61 @@
+import { SmartBuffer } from 'smart-buffer'
+import {
+  CCreateLoanScheme,
+  CreateLoanScheme
+} from '../../../../src/script/dftx/dftx_loans'
+import { OP_CODES } from '../../../../src/script'
+import { toBuffer, toOPCodes } from '../../../../src/script/_buffer'
+import { OP_DEFI_TX } from '../../../../src/script/dftx'
+import BigNumber from 'bignumber.js'
+
+it('should bi-directional buffer-object-buffer', () => {
+  const fixtures = [
+    '6a20446654784cc800000080b2e60e0000000006736368656d650000000000000000'
+  ]
+
+  fixtures.forEach(hex => {
+    const stack = toOPCodes(
+      SmartBuffer.fromBuffer(Buffer.from(hex, 'hex'))
+    )
+    const buffer = toBuffer(stack)
+    expect(buffer.toString('hex')).toStrictEqual(hex)
+    expect((stack[1] as OP_DEFI_TX).tx.type).toStrictEqual(0x4c)
+  })
+})
+
+const header = '6a20446654784c' // OP_RETURN(0x1a) OP_PUSHDATA1(0x20) CDfTx.SIGNATURE(0x44665478) CCreateLoanScheme.OP_CODE(0x4c)
```

Comment:
not OP_PUSHDATA1(0x20) but 0x20 is length. See getLenOpBuffer in `packages/jellyfish-transaction/src/script/data.ts`